src/pages/Weather/Weather.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import Card from '../../components/Card/Card';
+
+function Weather() {
```

Comment:
As the previous reviewers requested, its important that the Home page has a list of categories that could be filtered using given parameters, e.g. category name. Your project at its current state, does not count with this requirement. Please be sure to add this change so that your project is approved 🙌 You could filter by it by continent in the case of your app 🤓